### DISCUSSION TITLE
Again problem In Spark

### DISCUSSION THREAD

**Discussion by OmegaBeam69** [ROM-builders/temporary#21002] [General]
Can I have a Clean Build?
Recent error I faced:
https://cirrus-ci.com/task/6709262575992832

**Comment by tejas101k** [comment]
Learn Reading the errors.

This due the [outdated fork of caf display hal you have included in your manifest](https://github.com/OmegaBeam69/local_manifest/blob/c1f888c745ececdf04e4c2f59a8efe2660f3ed3a/local_manifest.xml#L22) Sync it with original source provided on lineage remote in your org manifest.

**Reply by OmegaBeam69** [reply]
I'm building with this repo for past 3-4 months

**Reply by tejas101k** [reply]
And what about the recent commits to fix your gralloc errors?
https://github.com/LineageOS/android_hardware_qcom_display/commit/ac823d9fd665e9831e20990a53512633fbfe8ee9

Ever cared about looking at recent commits joined in for the fixes?
